configure.ac

```diff
@@ -29,6 +29,11 @@ fi
     
 AC_PATH_TOOL(PKG_CONFIG, pkg-config)
 
+# no .pc for GMP
+AC_SEARCH_LIBS([__gmpz_init], [gmp])
+# pkg-config file Nettle doesn't provide hogweed in libs
+AC_SEARCH_LIBS([nettle_rsa_public_key_clear], [hogweed])
```

Comment:
Yep. Got the same feeling but it's the best I could come up with as pkg-config for nettle doesn't return -lhogweed (for --libs). My google-fu didn't help either:)
So if this is a real showstopper I could devote some time to try something more decent.